src/amqproxy/cli.cr

```diff
@@ -149,17 +76,22 @@ class AMQProxy::CLI
     unless server = @server
       raise "Can't call shutdown before run"
     end
+
+    unless config = @config
+      raise "Configuration has not been loaded"
```

Comment:
This means that for some reason `@config` isn't set we dont disconnect clients, we just stop the process. Isn't `@config` always set here?

Reply:
You need to make sure that it is set by being strict that it has a value, i.e. not letting it be nilable :)